cmake/KVIrc.nsi.cmake

```diff
@@ -55,7 +55,7 @@ Var LocalDir
 
 ;--------------------------------
 !insertmacro MUI_RESERVEFILE_LANGDLL
-ReserveFile "${NSISDIR}\Plugins\UserInfo.dll"
+ReserveFile "${NSISDIR}\Plugins\x86-unicode\UserInfo.dll"
```

Comment:
If this needed for both x86 and x86_64?

Reply:
It's needed on appveyor VS2017 image. If this breaks compatibility on other platform. I can create link on appveyor.